docs/usage/node-local-dns.md

```diff
@@ -32,5 +32,6 @@ It is worth noting that:
 
 - When migrating from IPVS to IPTables, existing pods will continue to leverage the node-local-dns cache. 
 - When migrating from IPtables to IPVS, only newer pods will be switched to the node-local-dns cache. 
+- Switching node-local-dns off by removing the annotation can be a rather destructive operation that might result in pods without a working dns configuration. We absolutely recommend to roll the cluster if you disable node-local-dns.
```

Comment:
> We absolutely recommend to roll the cluster if you disable node-local-dns.

How would this look like? 👀 What does an end user needs to do?

Reply:
I think my recommendation to "roll the cluster" is not helpful. I guess there is no way of disabling this in a cluster without having to rollout restart everything. But still this will cause a lot of trouble.
Are you fine with just telling people that they have to be careful with this one and not to give a suggestion what to do after switching it off? (I actually don't know what to do, just thinking rollout restarting everything should help).

Reply:
Sure, I'm with this for now if there is no better suggestion. That's already good information.